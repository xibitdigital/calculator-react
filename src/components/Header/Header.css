.header {
    background-color: #1795d4;
    color: white;
    padding: 10px;
}

.header__logo {
}

.header__text {
    padding: 5px 0;
}
