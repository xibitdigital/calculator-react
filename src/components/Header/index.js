import React from 'react'
import './Header.css'

const Header = () => (
    <header className="header">
        <div className="header__logo">
        </div>
        <div className="header__text">Sample calculator</div>
    </header>
)

export default Header
