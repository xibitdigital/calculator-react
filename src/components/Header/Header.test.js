import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import Header from '.'

test('Should render a Header', () => {
    const header = shallow(<Header />)

    expect(shallowToJson(header)).toMatchSnapshot()
})
