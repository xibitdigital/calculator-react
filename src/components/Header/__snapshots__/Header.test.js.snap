// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a Header 1`] = `
<header
  className="header"
>
  <div
    className="header__logo"
  />
  <div
    className="header__text"
  >
    Sample calculator
  </div>
</header>
`;
